tests/directory.rs

```diff
@@ -98,7 +98,7 @@ fn truncated_directory_in_root() -> io::Result<()> {
 #[test]
 #[ignore]
 fn git_repo_root() -> io::Result<()> {
-    let tmp_dir = TempDir::new()?;
+    let tmp_dir = TempDir::new_in(home_dir().unwrap())?;
     let repo_dir = tmp_dir.path().join("rocket-controls");
```

Comment:
Instead of creating `tmp_dir`, can't we just say that `let repo_dir = Path::new("$HOME/rocket-controls")`?

Reply:
To make the tests atomic, we use unique paths whenever we're performing filesystem operations that create or modify something.
Otherwise, having the directory already exist could cause collisions. For example, because we initialize the git repo, using the same directory to initialize a git repo again throws an error.

Reply:
Understand ... thanks for explaining. Seems fine to me then!